superbench/benchmarks/micro_benchmarks/hipblas_function.py

```diff
@@ -0,0 +1,120 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
Change the file name to "hipblaslt_function.py"

Reply:
updated, thanks